src/DotNetty.Transport/Internal/LoggingExtensions.cs

```diff
@@ -26,11 +26,19 @@
 using DotNetty.Common.Concurrency;
 using DotNetty.Common.Internal.Logging;
 using DotNetty.Transport.Channels;
+using DotNetty.Transport.Channels.Sockets;
 
 namespace DotNetty.Transport
 {
     internal static class TransportLoggingExtensions
-    {
+    {       
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public static void AbstractSocketIoCallbackSkipped<TChannel, TUnsafe>(this IInternalLogger logger, SocketChannelAsyncOperation<TChannel,TUnsafe> operation, TChannel channel, IEventLoop eventLoop) 
+            where TChannel : AbstractSocketChannel<TChannel, TUnsafe> where TUnsafe : AbstractSocketChannel<TChannel, TUnsafe>.AbstractSocketUnsafe, new()
+        {
+            logger.Debug($"Skipping callback schedule due to eventLoop closure and socket already closed. operation: {operation.LastOperation} / socketError: {operation.SocketError}; channel: {channel.Id} (IsOpen: {channel.IsOpen})");
```

Comment:
the eventLoop part of the log message is irrevelant.

Reply:
removed eventLoop as parameter

Reply:
and changed the log message